src/styles/layout.scss

```diff
@@ -69,7 +69,7 @@
   border-bottom: 1px solid var(--amplify-colors-neutral-20);
 }
 .layout-sidebar-menu {
-  padding: var(--amplify-space-large);
+  padding: var(--amplify-space-large) 0;
```

Comment:
Made the left/right padding be `0` so that the "hover" effect for the menu item spans the whole width of the menu.